src/olcs/OLCesium.ts

```diff
@@ -135,66 +173,41 @@ class OLCesium {
       this.canvas_.style['imageRendering'] = imageRenderingValue();
     }
 
-    this.canvas_.oncontextmenu = function() { return false; };
-    this.canvas_.onselectstart = function() { return false; };
+    this.canvas_.oncontextmenu = function () {
+      return false;
+    };
+    this.canvas_.onselectstart = function () {
+      return false;
+    };
 
     this.container_.appendChild(this.canvas_);
 
-    /**
-     * @type {boolean}
-     * @private
-     */
-    this.enabled_ = false;
-
-    /**
-     * @type {!Array.<ol.interaction.Interaction>}
-     * @private
-     */
-    this.pausedInteractions_ = [];
-
-    /**
-     * @type {?ol.layer.Group}
-     * @private
-     */
-    this.hiddenRootGroup_ = null;
-
-    const sceneOptions = options.sceneOptions !== undefined ? options.sceneOptions :
-      /** @type {Cesium.SceneOptions} */ ({});
-    sceneOptions.canvas = this.canvas_;
-    sceneOptions.scene3DOnly = true;
+    const sceneOptions: SceneOptions = options.sceneOptions !== undefined ?
+      {...options.sceneOptions, canvas: this.canvas_, scene3DOnly: true} :
+      {canvas: this.canvas_, scene3DOnly: true};
 
-    /**
-     * @type {!Cesium.Scene}
-     * @private
-     */
     this.scene_ = new Cesium.Scene(sceneOptions);
 
     const sscc = this.scene_.screenSpaceCameraController;
 
-    sscc.tiltEventTypes.push({
-      'eventType': Cesium.CameraEventType.LEFT_DRAG,
-      'modifier': Cesium.KeyboardEventModifier.SHIFT
-    });
+    if(Array.isArray(sscc.tiltEventTypes)) {
```

Comment:
Let's output a warning if it is not an array:

```suggestion
    if (!Array.isArray(sscc.tiltEventTypes)) {
      console.log('sscc is not an array');
    } else {
```